### PR TITLE
Fixed Windows CI: now systemd integration test is skipped on Windows

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by aaron-prindle** [comment]
This test hangs forever on windows as it opens a subshell that does not get closed.  Skipping this test on Windows will enable CI to work properly again.

File: test/integration/vm_systemd.go [hunk 1]
```diff
@@ -27,6 +28,9 @@ import (
 
 func testVMSystemd(t *testing.T) {
 	t.Parallel()
+	if runtime.GOOS == "windows" {
```
Comment (r2d4) [inline test/integration/vm_systemd.go]: Any clue why the test hangs?  Seems a bit odd to just skip it here

Comment (aaron-prindle) [inline test/integration/vm_systemd.go]: It is in above comment, it open a subshell that doesn't close but in other systems the output goes to the appropriate terminal and exits.

**Comment by aaron-prindle** [comment]
This is the output of the command:
gives this output:
[[0;1;31m0 loaded units listed.[[0m
To show all installed unit files use 'systemctl list-unit-files'.

~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
standard input[]

^ this is a prompt that you can type in, this shell does not close properly

File: test/integration/vm_systemd.go (hunk 1 above)
Comment (r2d4) [inline test/integration/vm_systemd.go]: Maybe we need to quote the command?  Or something special about the `--`

**Pull request merged**